### DISCUSSION TITLE
M-Audio firewire solo driver wanted

### DISCUSSION THREAD

**Discussion by 77colorz** [community/community#127211] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Greetings Friends
Im am new here and a noob, who desperatly needs the help of true masters who dwell in this community.

Is it possible for someone to develop a audio driver for my M-Audio firewire solo interface to work with my late 2011 15inch macbook pro .. 2.4 ghz .. 16 GB ram intel 3000 graphics running ventura 13.6.7 installed with the latest OPEN CORE PATCHER which is a miracle for me :) Thanx to the experts here. I am running Logic pro x 11 and am hoping someone would help me get the driver for this M-Audio firewire solo to work on my macbook .. i have atached a file i found on the net but it dont work with my setup .
[redacted by moderator]

.. Pls help me .. 

Best regards 
[redacted by moderator]

**Comment by ebndev** [comment]
Hi @77colorz,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this type of project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

I have also removed automatic downloads from your post and email, as we don't allow personal identifying information in Discussions. Thanks!
